python/cuml/benchmark/runners.py

```diff
@@ -245,8 +255,12 @@ def run_variations(
                     raise_on_error=raise_on_error,
                 )
                 for r in results:
+                    if "algo" in r:
```

Comment:
I'm not sure I understand this bit... it is appending r["algo"] to algo_name? I think that seems a little hacky. Maybe there's some other way to set the algo.name the way you want it up front?

Reply:
FIL takes an option `algo` which is one of `NAVIE, TREE_REORG, BATCH_TREE_REORG`. If we specify this `algo` option, the algorithm name which should be `FIL` will be overwritten by `NAVIE, TREE_REORG, BATCH_TREE_REORG`. 
Thus, I choose to append this option after the algorithm name

Reply:
Is it possible to use different names for those arguments, e.g. `'algo'` and `'method'`?

Reply:
Yes, that will be a better way. Do you mind if we rename the `algo` for FIL to be `method`, or maybe `fil_algo`?